WordPress/Classes/Extensions/URL+Helpers.swift

```diff
@@ -107,6 +107,10 @@ extension URL {
 
         return UTTypeConformsTo(uti as CFString, kUTTypeImage)
     }
+
+    var isGif: Bool {
+        return pathExtension == "gif"
+    }
```

Comment:
Would it make sense to do something like this:

```
    var isGif: Bool {
        if let uti = typeIdentifier {
            return UTTypeConformsTo(uti as CFString, kUTTypeGIF)
        } else {
            // default to inspecting the extension
            return pathExtension.lowercased() = "gif"
        }
    }
```

...checking the `typeIdentifier` if available, otherwise look in the `pathExtension` for a string match?

Reply:
Sounds good!